src/components/label/label.svelte

```diff
@@ -0,0 +1,67 @@
+<script context="module" lang="ts">
+  export const colors = [
+    'gray',
+    'secondary',
+    'primary',
+    'red',
+    'yellow',
+    'green',
+    'blue',
+    'purple',
+    'pink',
+    'teal',
+    'hero'
+  ] as const
+  export type Color = typeof colors[number]
+
+  export const modes = ['faint', 'heavy'] as const
```

Comment:
`modes` should be renamed to `'loud' | 'default'` cc @aguscruiz

Reply:
Renamed - defaults to `default`